command_line/stills_process.py

```diff
@@ -240,17 +241,27 @@ def do_import(filename, load_models=True):
 
     from dxtbx.imageset import ImageSetFactory
 
+    all_experiments = ExperimentList()
     for experiment in experiments:
-        if load_models:
-            experiment.load_models()
+        # Convert from ImageSequence to ImageSet, if needed
         imageset = ImageSetFactory.imageset_from_anyset(experiment.imageset)
-        imageset.set_scan(None)
-        imageset.set_goniometer(None)
-        experiment.imageset = imageset
-        experiment.scan = None
-        experiment.goniometer = None
+        for i in range(len(imageset)):
+            # Preserve original models if they were available (in the case of an image file
+            # they will not be, but in the case of a previously processed experiment list,
+            # then they may be available
+            expt = Experiment(
+                imageset=imageset[i : i + 1],
```

Comment:
This will preserve whether the input is an `ImageSet` or `ImageSequence` so represents a behaviour change?

Reply:
The `imageset_from_anyset` call converts an `ImageSequence` to an `ImageSet`, which (especially with #1148) will mean the data are treated as stills downstream.